src/Contracts/ActionContracts.ts

```diff
@@ -4,6 +4,7 @@
 export enum TabKind {
   SQLDocuments,
   MongoDocuments,
+  MongoSchema,
```

Comment:
>MongoSchema [](start = 2, length = 11)

want to call it as Schema sice we may be able to extend it outside of mongoAPI as well

Reply:
Since there's already a `Schema` tab we need to think of a different name. I can rename it once we decide on a better name and start adapting for other API types.

Reply:
Renaming this to `SchemaAnalyzer` instead for now.